Chapters/Chapter1.md

```diff
@@ -1,5 +1,163 @@
+  "Lovely day"
 
-You cannot hear the first note of a jiyu's song. There is no grand entrance or catchy hook. It's more like a choir of thousands of distant voices all entering and exiting at different times with their pre-ordained notes. When you start to hear the song, you realize that it's been singing for a while, only your mind was not clear enough to focus on it. No mind has been or will ever be clear enough to know which of a jiyu's leaves sang first. Only the tree can know, and it will forever keep it secret.
+  Startled, I sat up to see a silhouette of Melna standing in the light, her hair dancing in time with the leaves.
 
-It was under this tree that I was born. Among the plants and animates that called this magnificent tree home, I made my first cry to the heavens and the heavens sang back through the leaves. My mother used to tell me it was the most beautiful thing watching me be born. They had been waiting all night for me to start inching out, but I wouldn't budge until just before dawn. It had been raining that night and the leaves of the tree were glistening with a crescent of green light from our sibling planet, Surien. She told me that the song the tree sang that morning had never sounded quite so much like a lullaby, carrying me in its arms the way she would do. I still imagine it sometimes when I come here, laying in her arms within the greatest sound in all the world.
+  "How long have you been here?" I asked.
+
+  "Just a click. I figured you would be here today."
+
+  I stayed silent, still listening for the leaves and my mother's voice.
+
+  "You've been gone a while. Your breakfast is cold."
+
+  "I'll be there in a bit."
+
+  She stared at me for a moment - I assume to size up my strange behavior, and then turned to walk back home.
+
+  She reassured me, "I feel it too. There's a chill in the air."
+
+  I felt no comfort.
+
+
+
+*************
+
+
+
+  Kip was at the stove reheating my breakfast and reading a postletter with his morning face on. His hair, the furrowed nest of a bird that took a liking to crooked branches. He looked up, locking his tired eyes to mine, and put on his goofiest morning smile which only complimented his hair. I walked up to nuzzle him. Melna was squatting at the eating table, reading her daily missive from the provincial council.
+
+  "Who's the letter from?", I asked Kip casually.
+
+  "My parents. They're on vacation to the east bay this week."
+
+  "That's a strange place to vacation," said Melna.
+
+  "Yea. They've always been ones to go on adventures to exotic places. They were really wanting to see the sky when Ara is high. Something about brightness meditation."
+
+  I was immediately excited. "Oh, I've heard about that. They say there are no clouds or fog and the whole sky is bright blue like the sunrise. Everything is so hot that you sweat out all your worries."
```

Comment:
What about, "I chimed in excitedly, ..." show-don't-tell rule of thumb. Let your character's emotional state slip through, because most of the time we aren't actually aware of our emotional states.